File: C/3-printing-quotes.c
#include <stdio.h>
#include <string.h>

int main(int argc, char *argv[]) {
    char quote[64], author[64];

    printf("What is the quote? ");
    if (fgets(quote, sizeof(quote), stdin))
        quote[strlen(quote) - 1] = '\0';
    
    printf("Who said it? ");
    if (fgets(author, sizeof(author), stdin))
        author[strlen(author) - 1] = '\0';

    printf("%s says, \"%s\"\n", author, quote);

    return 0;
}

File: go/3-printing-quotes.go
package main

import "fmt"

func main() {
	var quote, author string

	fmt.Printf("What is the quote? ")
	fmt.Scanln(&quote)
	fmt.Printf("Who said it? ")
	fmt.Scanln(&author)

	fmt.Printf("%s says, \"%s\"\n", author, quote)
}